backend/app/cmd/server.go

```diff
@@ -211,6 +211,10 @@ type NotifyGroup struct {
 		VerificationSubject string `long:"verification_subj" env:"VERIFICATION_SUBJ" description:"verification message subject"`
 		AdminNotifications  bool   `long:"notify_admin" env:"ADMIN" description:"notify admin on new comments via ADMIN_SHARED_EMAIL"`
 	} `group:"email" namespace:"email" env-namespace:"EMAIL"`
+	Slack  struct {
+		Token   string        `long:"token" env:"TOKEN" description:"slack token"`
+		Channel string        `long:"chan" env:"CHAN" description:"slack channel"`
```

Comment:
If I am not mistaken, a channel is not a channel name but some unique ID, if it's the case I would propose to add a note both here and in the Readme